third_party/meerkat/Component/mmSH/android/java/src/com/samsung/android/meerkat/MeerkatServerService.java

```diff
@@ -121,8 +137,14 @@ public void onDestroy() {
 
     @Override
     public void onSharedPreferenceChanged(SharedPreferences prefs, String key) {
-        if (key.equals("capability")) {
-            setCapability(prefs.getString(key, ""));
+        Log.i(TAG, "key : " + key + ", value : " + prefs.getString(key, ""));
+        try {
+            JSONParser parser = new JSONParser();
+            JSONObject jsonObj = (JSONObject) parser.parse(prefs.getString(key, ""));
+            Log.i(TAG, "id : " + (String) jsonObj.get("id") + ", name : " + (String) jsonObj.get("name") + ", features : " + (String) jsonObj.get("features"));
```

Comment:
This log seems to have considered only offload.js, how about logging just key and value?

Reply:
Done.